.github/workflows/release.yml

```
@@ -0,0 +1,110 @@
+      with:
+        path: ~/.m2
+        key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
+        restore-keys: ${{ runner.os }}-m2
+        
+    - name: Build with Maven
+      run: |
+        mvn clean package assembly:single
+        ls -la target/
+        
+    - name: Create Release Assets
+      run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2035:info:11:8: Use ./*glob* or -- *glob* so names with dashes won't become options

(shellcheck)